src/node/build/index.ts

```diff
@@ -621,7 +621,7 @@ async function doBuild(options: Partial<BuildConfig>): Promise<BuildResult[]> {
                 await fs.emptyDir(outDir)
               }
               if (emitIndex) {
-                await fs.writeFile(emitIndexPath, result.html)
+                await fs.writeFile(emitIndexPath!, result.html)
```

Comment:
I prefer to return an empty string from `getEmitIndexPath` so these aren't necessary.